application/src/test/resources/speedtest/travelSearch-expected-results-bd.csv

```diff
@@ -1,3 +1,3 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,41m59s,0,0,12:00:00,12:41:59,,,,,Unknown transit 0tx 41m59s
-2,0,41m59s,0,0,12:00:00,12:41:59,,,,,Unknown transit 0tx 41m59s
+1,0,41m56s,0,0,12:00:00,12:41:56,,,,,Unknown transit 0tx 41m56s
+2,0,41m56s,0,0,12:00:00,12:41:56,,,,,Unknown transit 0tx 41m56s
```

Comment:
Do you know why all of these speed tests are 3 seconds quicker? Ideally changing the internal angle representation should not change travel time.

Reply:
It is because of the inAngle / outAngle fix for creating the temporary edge has changed the turn time near the origin / destination.